Automatic dependency update workflow fails in shell script

See job https://github.com/exasol/glue-connector/actions/runs/9377098985/job/25818124414

Step "Generate Pull Request Comment" fails with message
```
/home/runner/work/_temp/c4127951-5484-4bde-9655-76513a496e49.sh: line 17: unexpected EOF while looking for matching `''
Error: Process completed with exit code 2.
Error: Unable to process file command 'output' successfully.
Error: Invalid value. Matching delimiter not found 'EOF'
```

This is caused by the script containing an unmatched `'` in line

```sh
echo 'Due to restrictions workflow `dependencies_update.yml` can't update other workflows, see https://github.com/exasol/project-keeper/issues/578 for details.' >> "$GITHUB_OUTPUT"
```

--- a/.github/workflows/dependencies_update.yml
+++ b/.github/workflows/dependencies_update.yml
@@ -82,7 +82,7 @@ jobs:
           echo >> "$GITHUB_OUTPUT"
           echo '# ⚠️ Notes ⚠️' >> "$GITHUB_OUTPUT"
           echo '## Run PK fix manually' >> "$GITHUB_OUTPUT"
-          echo 'Due to restrictions workflow `dependencies_update.yml` can't update other workflows, see https://github.com/exasol/project-keeper/issues/578 for details.' >> "$GITHUB_OUTPUT"
+          echo 'Due to restrictions workflow `dependencies_update.yml` cannot update other workflows, see https://github.com/exasol/project-keeper/issues/578 for details.' >> "$GITHUB_OUTPUT"
           echo 'Please checkout this PR locally and run `mvn com.exasol:project-keeper-maven-plugin:fix --projects .`' >> "$GITHUB_OUTPUT"
           echo '## This PR does not trigger CI workflows' >> "$GITHUB_OUTPUT"
           echo 'Please click the **Close pull request** button and then **Reopen pull request** to trigger running checks.' >> "$GITHUB_OUTPUT"

--- a/doc/changes/changelog.md
+++ b/doc/changes/changelog.md
@@ -1,5 +1,6 @@
 # Changes
 
+* [4.3.3](changes_4.3.3.md)
 * [4.3.2](changes_4.3.2.md)
 * [4.3.1](changes_4.3.1.md)
 * [4.3.0](changes_4.3.0.md)

--- a/doc/changes/changes_4.3.3.md
+++ b/doc/changes/changes_4.3.3.md
@@ -1,0 +1,59 @@
+# Project Keeper 4.3.3, released 2024-06-05
+
+Code name: Fix bug in automatic dependency update workflow
+
+## Summary
+
+This release fixes a bug in a shell script of the automatic dependency update workflow that caused the workflow to fail with error message
+
+```
+line 17: unexpected EOF while looking for matching `''
+```
+
+## Bugfixes
+
+* #580: Fixed bug in automatic dependency update workflow
+
+## Dependency Updates
+
+### Project Keeper Core
+
+#### Compile Dependency Updates
+
+* Updated `com.exasol:project-keeper-shared-model-classes:4.3.2` to `4.3.3`
+
+#### Runtime Dependency Updates
+
+* Updated `com.exasol:project-keeper-java-project-crawler:4.3.2` to `4.3.3`
+
+#### Test Dependency Updates
+
+* Updated `com.exasol:project-keeper-shared-test-setup:4.3.2` to `4.3.3`
+
+### Project Keeper Command Line Interface
+
+#### Compile Dependency Updates
+
+* Updated `com.exasol:project-keeper-core:4.3.2` to `4.3.3`
+
+#### Test Dependency Updates
+
+* Updated `com.exasol:project-keeper-shared-test-setup:4.3.2` to `4.3.3`
+
+### Project Keeper Maven Plugin
+
+#### Compile Dependency Updates
+
+* Updated `com.exasol:project-keeper-core:4.3.2` to `4.3.3`
+
+### Project Keeper Java Project Crawler
+
+#### Compile Dependency Updates
+
+* Updated `com.exasol:project-keeper-shared-model-classes:4.3.2` to `4.3.3`
+
+### Project Keeper Shared Test Setup
+
+#### Compile Dependency Updates
+
+* Updated `com.exasol:project-keeper-shared-model-classes:4.3.2` to `4.3.3`

--- a/parent-pom/pom.xml
+++ b/parent-pom/pom.xml
@@ -28,7 +28,7 @@
         </repository>
     </distributionManagement>
     <properties>
-        <revision>4.3.2</revision>
+        <revision>4.3.3</revision>
         <!-- Integration test ProjectKeeperMojoIT starts a Maven build which requires Java 17. -->
         <java.version>17</java.version>
         <maven.version>3.9.7</maven.version>

--- a/project-keeper/src/main/resources/templates/.github/workflows/dependencies_update.yml
+++ b/project-keeper/src/main/resources/templates/.github/workflows/dependencies_update.yml
@@ -78,7 +78,7 @@ jobs:
           echo >> "$GITHUB_OUTPUT"
           echo '# ⚠️ Notes ⚠️' >> "$GITHUB_OUTPUT"
           echo '## Run PK fix manually' >> "$GITHUB_OUTPUT"
-          echo 'Due to restrictions workflow `dependencies_update.yml` can't update other workflows, see https://github.com/exasol/project-keeper/issues/578 for details.' >> "$GITHUB_OUTPUT"
+          echo 'Due to restrictions workflow `dependencies_update.yml` cannot update other workflows, see https://github.com/exasol/project-keeper/issues/578 for details.' >> "$GITHUB_OUTPUT"
           echo 'Please checkout this PR locally and run `mvn com.exasol:project-keeper-maven-plugin:fix --projects .`' >> "$GITHUB_OUTPUT"
           echo '## This PR does not trigger CI workflows' >> "$GITHUB_OUTPUT"
           echo 'Please click the **Close pull request** button and then **Reopen pull request** to trigger running checks.' >> "$GITHUB_OUTPUT"

